geps/gep-1619.md

```diff
@@ -88,16 +235,14 @@ We can also examine how session persistence and session affinity functionally wo
 This tiered decision-based logic is consistent with the idea that session persistence is an exception to load balancing. Though there are different ways to frame this relationship, this design will influence the separation between persistence and affinity API design.
 
 ### Implementations
-In this section, we will describe how implementations achieve session persistence and affinity, along with a breakdown of related configuration options. Input from implementations is appreciated to complete this information.
+In this section, we will describe how implementations achieve session persistence, along with a breakdown of related configuration options. Input from implementations is appreciated to complete this information.
 
 In the following tables, we will example two types of APIs:
```

Comment:
did you intend to say "we will have examples of two types of APIs"

Reply:
Yes, I was trying to categorize the implementations into two types, as the text mentions:
```
In the following tables, we will example two types of APIs:
1. Dataplane APIs
2. Implementation APIs
```

Dataplanes == Envoy, Haproxy, Nginx
Implementations == Contour, Istio, Envoy Gateway, Haproxy Ingress...

Does that make sense?

Reply:
Yes, all good.